src/app/common/validation/forms/amount-validators.ts

```diff
@@ -79,19 +79,22 @@ export function stxAmountValidator() {
     .concat(stxAmountPrecisionValidator(formatPrecisionError()));
 }
 
-export function stxAvailableBalanceValidator(availableBalance: Money) {
+export function stxAvailableBalanceValidator(
+  availableBalance: Money,
+  pendingTxsBalance: BigNumber
```

Comment:
Can we also pass this in as `Money`, rather than just the `BigNumber`?

Thinking behind this suggestion is to keep the core data models whole. This enables certain properties. One example: a safer way to perform this operation might be to create a separate helper fn `subtractMoney(x: Money, y: Money)` that throws an error `if (x.symbol !== y.symbol)`